lib/epics/xe2.rb

```diff
@@ -0,0 +1,5 @@
+class Epics::XE2 < Epics::CDD
```

Comment:
if we go for inheritance here, I think XE2 should inherit from CCT, given that it is uploading pain.001 files

Reply:
For sure, good point! I will change it

Reply:
Changed